cardano-submit-api/src/Cardano/TxSubmit/Types.hs

```diff
@@ -50,34 +57,40 @@ data TxSubmitWebApiError
   | TxSubmitBadTx !Text
   | TxSubmitFail TxCmdError
 
-newtype EnvSocketError = CliEnvVarLookup Text deriving (Eq, Show)
+deriving instance Generic TxSubmitWebApiError
```

Comment:
I'd rather we take the time to be explicit about `ToJSON TxSubmitWebApiError`. It gives us better control over deprecation handling or in general customizing the output to our liking.

Reply:
I can certainly expand this into a manual `ToJSON` instance.  Do you want the `ToJSON` instance to behave like the generic instance or something different?

Reply:
I don't have a preference as long as it's explicit